Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -111,13 +111,15 @@ buildRideListItem QRide.RideItem {..} = do
 
 getActualRoute :: MonadFlow m => Common.DriverEdaKafka -> m Common.ActualRoute
 getActualRoute Common.DriverEdaKafka {..} =
-  case (lat, lon, ts) of
-    (Just lat_, Just lon_, ts_) -> do
+  case (lat, lon, ts, acc, rideStatus) of
+    (Just lat_, Just lon_, ts_, Just acc_, Just rideStatus_) -> do
       lat' <- readMaybe lat_ & fromMaybeM (InvalidRequest "Couldn't find driver's location.")
       lon' <- readMaybe lon_ & fromMaybeM (InvalidRequest "Couldn't find driver's location.")
+      acc' <- readMaybe acc_ & fromMaybeM (InvalidRequest "Couldn't find accuracy")
```

Comment:
Note that this will throw error in the API even if any of these fields is missing for one record(out of thousands of records). RideStatus can be made mandatory but is accuracy won't be present in each record